docs/docusaurus.config.ts

```diff
@@ -221,16 +209,9 @@ const config: Config = {
             },
           ],
         },
-        {
-          title: "Github",
-          items: [
-            {
-              label: "Weave",
-              href: "https://github.com/wandb/weave",
-            },
-          ],
-        },
       ],
+    },
+    footer: {
       copyright: `Made with ❤️ by Weights & Biases`,
```

Comment:
No copyright year?

Reply:
it did not exist prior. This pr is about modifying the docs styling rather than content adjustments